SLHCUpgradeSimulations/Configuration/python/phase2TkFlat.py

```diff
@@ -27,6 +29,45 @@ def customise(process):
 
     return process
 
+
+def customise_DigiTkOnly(process):
```

Comment:
please move this function somewhere in L1 test areas. Its not something for general use. (or it if is, lets discuss that)